pkgs/applications/science/misc/colmap/default.nix

```diff
@@ -37,7 +37,7 @@ mkDerivation rec {
   nativeBuildInputs = [
     cmake
   ] ++ lib.optionals cudaSupport [
-    cudaPackages.autoAddOpenGLRunpathHook
+    cudaPackages.autoAddDriverRunpathHook
```

Comment:
One last cosmetic thing, and I'm only bringing this up because we're changing a name which we don't do often: do we really want to keep the `Hook` suffix around?..

Grepping for `makeSetupHook {` I see there's a bunch of examples (unfamiliar to me) which don't follow the convention: `makeGCOVReport`, `moveBuildTree`, `useOldCXXAbi`, `copyDesktopItems`, `setJavaClassPath`, etc, etc...


Should we simply make this `autoAddDriverRunpath`?

Reply:
I agree that `autoAddDriverRunpath` is IMHO already explicit enough (especially the autoAdd part) that we can get rid of the `Hook`. Another solution is to scope that within cudaPackages: `cudaPackages.hooks.autoAddDriverRunpath`, for example. I'm also fine having staying it the way it is right now, it's a bit verbose but at least not misleading.

Reply:
> cudaPackages.hooks

I don't think so, I think the next logical step will be to move `autoAddDriverRunpath(Hook)` up from `cudaPackages` to `pkgs`

Reply:
So, do you want me to remove the `Hook` part?

Reply:
Ok yes please, let's do this.